tasks/memory-game/group-Row1/src/MemoryGame.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Created by chelo on 9/17/2016.
+ */
+
+ var MemoryGame = function (size){
+	this.size = size;
+	this.x = size;
+	this.y = size;
+	this.array = new Array();
+};
+
+/**
+ * Returns array values
+ * @return {Array} array
+ */
+MemoryGame.prototype.getArray = function () {
+	for (var i = 0; i < this.x; i++){
```

Comment:
This was thought to have a new version in the future to have a rectangle instead a square ;-)